src/components/organisms/Footer/index.tsx

```diff
@@ -7,13 +7,53 @@ const Wrapper = styled.footer`
   padding: 16px 0;
   font-size: 12px;
   background-color: ${colors.White};
-  display: flex;
-  align-items: center;
-  justify-content: center;
+  ul {
+    justify-content: start;
+    column-count: 3;
+    display: flex;
+    list-style: none;
+  }
+  li {
+    padding-right: 4px;
+    padding-left: 4px;
+  }
+  li + li {
+    list-style: "|";
+  }
+  @media screen and (min-width: 1152px) {
+    li {
+      padding-right: 8px;
+      padding-left: 8px;
+    }
+  }
```

Comment:
[may] `li` そのものに `padding-left` がつくと、行の初めの要素にも余白ができてしまい、厳密にはデザイン通りとはならないです。
![スクリーンショット 2022-09-06 14 29 29](https://user-images.githubusercontent.com/30136823/188554448-a31c4a68-daf6-431a-b62d-b69f28e8c8b2.png)
![スクリーンショット 2022-09-06 14 29 32](https://user-images.githubusercontent.com/30136823/188554452-990ffc6e-6125-4f2a-a051-f8a81c2af5bf.png)

また、 `list-style` で区切ると、余白が微妙に異なって見えるので、 `before` 擬似要素に `content` として入れて、そこに余白をもたせるという方法も良さそうです。

それぞれを合わせると、具体的には以下のようにすると、綺麗に見えそうです。

```suggestion
  li + li {
    padding-left: 4px;
    &::before {
      padding-right: 4px;
      content: '|';
    }
  }
  @media screen and (min-width: 1152px) {
    li + li {
      padding-left: 4px;
      &::before {
        padding-right: 4px;
      }
    }
  }
```